src/Glpi/Controller/KBController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller;
+
+use Glpi\Exception\Http\AccessDeniedHttpException;
+use Glpi\Exception\Http\NotFoundHttpException;
+use KnowbaseItem;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+final class KBController extends AbstractController
+{
+    #[Route(
+        "/Knowbase/KnowbaseItem/{knowbaseitems_id}/Content",
+        name: "knowbaseitem_content",
+        requirements: [
+            'knowbaseitems_id' => '\d+',
+        ]
+    )]
+    public function getArticleContent(Request $request): Response
```

Comment:
The route path should match the controller namespace/class name/method name.

For instance, if your route is `/Knowbase/KnowbaseItem/{knowbaseitems_id}/Content`, then the method path should be `\Glpi\Controller\Knowbase\KnowbaseItemController::content()`.

With this, it would be easier to locate the corresponding file as it would not require to do a text search to find the route declaration.